src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+/* code here */
+@import "../utils/mixins";
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
```

Comment:
Use color variable

Reply:
![image](https://user-images.githubusercontent.com/110053436/218108059-a0355e86-28ed-4a68-8110-619aa27d5228.png)

why? i just once used this